src/test/java/org/kiwiproject/dropwizard/util/admin/StandardAdminConfiguratorTest.java

```diff
@@ -0,0 +1,197 @@
+        }
+    }
+
+    @Nested
+    class AddMetrics {
+
+        @Test
+        void shouldAddServerLoadGauge() {
+            var environment = DropwizardMockitoMocks.mockEnvironment();
+
+            var configurator = StandardAdminConfigurator.builder()
+                    .environment(environment)
+                    .registerExpiringKeystoreHealthCheck(false)
+                    .build();
+
+            configurator.addMetrics();
+
+            verify(environment.metrics()).register(eq(ServerLoadGauge.NAME), any(ServerLoadGauge.class));
+        }
+    }
+
+    @Nested
+    class AddHealthChecks {
+
+        @Test
+        void shouldAddHttpConnectionsHealthCheck() {
+            var environment = DropwizardMockitoMocks.mockEnvironment();
+
+            var configurator = StandardAdminConfigurator.builder()
+                    .environment(environment)
+                    .registerExpiringKeystoreHealthCheck(false)
+                    .build();
+
+            configurator.addHealthChecks();
+
+            verify(environment.healthChecks()).register(eq(HttpConnectionsHealthCheck.DEFAULT_NAME), any(HttpConnectionsHealthCheck.class));
+        }
+
+        @Test
+        void shouldNotAddExpiringKeystoreHealthCheck_WhenNotConfigured() {
+            var environment = DropwizardMockitoMocks.mockEnvironment();
+
+            var configurator = StandardAdminConfigurator.builder()
+                    .environment(environment)
+                    .registerExpiringKeystoreHealthCheck(false)
+                    .build();
+
+            configurator.addHealthChecks();
+
+            verify(environment.healthChecks(), times(0)).register(any(String.class), any(ExpiringKeystoreHealthCheck.class));
+        }
+
+        @Test
+        void shouldAddExpiringKeystoreHealthCheck_ForKeyStoreAndTrustStore_WhenConfigured() {
+            var environment = DropwizardMockitoMocks.mockEnvironment();
+            var tlsConfiguration = TlsContextConfiguration.builder()
+                    .keyStorePath("/path/to/keystore")
+                    .keyStorePassword("password")
+                    .trustStorePath("/path/to/truststore")
+                    .trustStorePassword("password")
+                    .build();
+
+            var configurator = StandardAdminConfigurator.builder()
+                    .environment(environment)
+                    .tlsConfiguration(tlsConfiguration)
+                    .build();
+
+            configurator.addHealthChecks();
+
+            verify(environment.healthChecks()).register(eq("Key store"), any(ExpiringKeystoreHealthCheck.class));
+            verify(environment.healthChecks()).register(eq("Trust store"), any(ExpiringKeystoreHealthCheck.class));
+        }
+
+        @Test
+        void shouldAddExpiringKeystoreHealthCheck_ForTrustStoreOnly_WhenConfiguredAndKeyStorePathIsMissing() {
+            var environment = DropwizardMockitoMocks.mockEnvironment();
```

Comment:
I think this line is in every single test except the very first one...worth extracting to a field and initializing in a `@BeforeEach setUp()` method?

Reply:
Rewrote all of this